src/views/Auction/components/Positions.tsx

```diff
@@ -23,9 +26,11 @@ const StyledSwiper = styled.div`
     align-items: center;
     padding-bottom: 25px;
     padding-top: 25px;
+    gap: ${gap};
```

Comment:
can you just use the `style` prop to set `marginRight` in the `SwiperSlide` component